cipher/diffiehellman/diffiehellmankeyexchange_test.go

```diff
@@ -39,3 +39,21 @@ func TestDiffieHellmanKeyExchange(t *testing.T) {
 		}
 	})
 }
+
+func FuzzGenSharedKey(f *testing.F) {
+	f.Add(int64(13))
+	f.Fuzz(func(t *testing.T, prvKey int64) {
+		// explicitly ignoring the return error
+		// here we want to test that no strange behaviors are raised when executing GenerateShareKey
+		_ = GenerateShareKey(prvKey)
+	})
```

Comment:
This is unnecessary I think... Is there a specific reason why fuzzing is even needed here if we are not checking the error?

Reply:
Actually I wrote it wrong: it's not return error but the return value. I can update that if you agree with it.

Reply:
I didn't understand what you meant, but please update it so that I can see what you mean and we can make progress on this further 😄